CONTRIBUTING.md

```diff
@@ -187,6 +187,11 @@ repository, we suggest doing some cleanup of these branches.
   $ git push {username} :<branchname>
   ```
 
+## Contributing Documentation
+
+The documentation for each repository is available in the "**docs/**" directory of that repository, is written in [Markdown format], and is built using [MkDocs].
```

Comment:
The referenced links for `Markdown format` and `MkDocs` are missing here.

Reply:
Thanks for spotting that. Just added them in b01aca3.